### ISSUE TITLE
Extract Type Information from Symbol

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#1558]
This was brought up in our Drasil meeting yesterday when discussing Projectile. We should be able to "extract" some typing information from the symbol, in particular, vector information. Vectors (ie. quantities with bolded symbols, although potential overlap with tensors was brought up) would be printed with "vector" after the quantity when using `phrase`. For example, if `accel` is defined with the term `"acceleration"` and the symbol `vec lA`, `phrase accel` would output "acceleration vector". @smiths and @Mornix should be able to fill in any gaps in this issue.

**Comment by smiths** [comment]
Further to what @samm82 said, I believe we all thought it was a good idea to automatically generate the word vector to go along with the phrase.  The conversation was started with my request to @samm82 to add the word vector to distinguish the scalar acceleration (symbol a) from the vector acceleration (symbol \bf{a}).  The motivation for this question is to see if you anticipate any unwanted surprises from making this change.

**Comment by JacquesCarette** [comment]
I completely agree with having our chunks carry some type information that can be queried. If this isn't already possible (I think it is), it ought to be.

Whether we want `phrase` to always use the type information to insert some words, that's less clear to me. That we want a combinator that does that? Most definitely. But my feeling is that modifying `phrase` itself is not going to do what you really want.
